CorsixTH/Src/th_gfx_common.h

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2009-2013 Peter "Corsix" Cawley and Edvin "Lego3" Linge
+Copyright (C) 2012-2013 Luis "Driver" Duarte and Robert "flackr" Flack
```

Comment:
2012-2013?

Reply:
Ah, I got the mistaken idea from 2009-2013 in many files that the year here was not related to the files but instead the contributors.